src/decorator/decoratorProps.ts

```diff
@@ -1,48 +1,67 @@
-interface TogglesConfig {
-  visVeileder: boolean;
+export interface DecoratorProps {
+  enhet?: string | undefined; // Konfigurasjon av enhet-kontekst
+  accessToken?: string | undefined; // Manuell innsending av JWT, settes som Authorization-header. Om null sendes cookies vha credentials: 'include'
+  fnr?: string | undefined; // Konfigurasjon av fødselsnummer-kontekst
+  userKey?: string | undefined; // Om man ikke ønsker å bruke fnr i urler, kan andre apper kalle contextholder for å generere en midlertidig kode. Hvis App A skal navigere til App B som har dekoratøren, må App A først sende en post request til /fnr-code/generate med {fnr: string} i bodyen, dette returnerer {fnr: string, code: string} til App A. App A kan så navigere til App B og sende med denne koden. App B kan så sende den koden inn til dekoratøren i userKey  propen og så henter dekoratøren fnr for den koden fra contextholderen.
+  enableHotkeys?: boolean | undefined; // Aktivere hurtigtaster
+  fetchActiveEnhetOnMount?: boolean | undefined; // Om enhet er undefined fra container appen, og denne er satt til true, henter den sist aktiv enhet og bruker denne.
+  fetchActiveUserOnMount?: boolean | undefined; // Om fnr er undefined fra container appen, og denne er satt til true for at den skal hente siste aktiv fnr.
+  onEnhetChanged: (enhetId?: string | null, enhet?: Enhet) => void; // Kalles når enheten endres
+  onFnrChanged: (fnr?: string | null) => void; // Kalles når fnr enheten endres
+  onLinkClick?: (link: { text: string; url: string }) => void; // Kan brukes for å legge til callbacks ved klikk på lenker i menyen. Merk at callbacken ikke kan awaites og man må selv håndtere at siden lukkes. Nyttig for å f.eks tracke navigasjon events i amplitude
+  appName: string; // Navn på applikasjonen
+  hotkeys?: Hotkey[]; // Konfigurasjon av hurtigtaster
+  markup?: Markup; // Egen HTML
+  showEnheter: boolean; // Vis enheter
+  showSearchArea: boolean; // Vis søkefelt
+  showHotkeys: boolean; // Vis hurtigtaster
+  environment: Environment; // Miljø som skal brukes.
+  urlFormat: UrlFormat; // URL format
+  proxy?: string | undefined; // Manuell overstyring av urlene til BFFs. Gjør alle kall til relativt path hvis true, og bruker verdien som domene om satt til en string. Default: false
 }
 
-interface Markup {
-  etterSokefelt?: string;
+export interface Markup {
+  etterSokefelt?: string; // Gir muligheten for sende inn egen html som blir en del av dekoratøren
```

Comment:
Kanskje noe vi vil benytte oss av for å feks søke opp person på navn eller fødselsdato? 👀

Reply:
Ja, det kunne vært noe :+1: